docker-compose.yml

```diff
@@ -581,7 +581,7 @@ services:
     ports:
       - "${TEST_MONGODB_PORT}:27017"
   mysql:
-    image: mysql:5.6
+    image: mysql:8
     platform: linux/x86_64
```

Comment:
I think we should remove the `platform` attribute to allow docker to pick up different architecture base on host machine.

Reply:
Uh, right. Oopsie! Will do!

Reply:
Fixed in 8a12f865083f83aba3d39e822232902bbebee8d8